sdcm/cluster.py

```diff
@@ -813,8 +813,12 @@ def start_network_interface(self, interface_name="eth1"):
             startup_interface_command = "ip link set {} up"
         self.remoter.sudo(startup_interface_command.format(interface_name))
 
+    @cached_property
```

Comment:
Will it serve as expected upgrading a node from some enterprise version like `2024.2.x` to OSS `2025.1.x` having the cache enabled?

Reply:
I think @vponomaryov has a point here, we should cache here.

`scylla_version` is cached and handled already

Reply:
I left it as it was before. We cached version so this one caching makes no difference

Reply:
it's being handled in:
```
    def forget_scylla_version(self) -> None:
        self.__dict__.pop("scylla_version_detailed", None)
        self.__dict__.pop("scylla_version", None)
        self.__dict__.pop('is_enterprise', None)
```

but I think we can remove the cache now that it's directly depends on scylla_version

Reply:
valid point, removed the cache from is_enterprise